roles/lib_openshift/molecule/origin-latest/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Need to add ansible-doc testing for modules, not just in lib_openshift but for all modules.